test/testnet/localnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Remote Version of Localnet
+
+Remote version of the test network generated by running:
+
+```bash
+> make localnet-start
+```
+
+from the Tendermint repository. There are a few additional minor tweaks, however:
+
+1. The proxy app is preconfigured to `kvstore`
+2. Logging format is JSON
+3. `persistent_peers` is preconfigured
+4. `create_empty_blocks` is set to `false`
+5. `prometheus` is set to `true` to allow for collecting Prometheus metrics
+
+## Contents
+This folder contains the relevant prerequisites to deploy this test network to a
+set of predefined hosts. Here is a description of a subset of the layout of this
+folder's contents:
+
+```
+|_ Makefile           The primary entrypoint to interacting with the test network.
+|_ requirements.txt   Python requirements for running the deployment scripts.
+|_ config             Ansible configuration management.
+   |_ hosts           The inventory file for the hosts to which we will be deploying the network.
```

Comment:
Storing hosts is also something to discuss. TNT is a quick and dirty, short-term solution to the problem of self-managed testnets. We're definitely not going to keep updating a hosts file on a repo when we create new testnets because testnet creation should (and will) be a more flexible solution.

To be honest, I'm beginning to think, this solution is not ready to be merged into the tendermint repo and we should have more testing, use-cases and discussions around its usage.